### PR TITLE
Add new tool: ConFindr

### PR DESCRIPTION
**Pull request by lowandrew**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This recipe adds [ConFindr](https://lowandrew.github.io/ConFindr/), a tool for detecting intra- or inter-species bacterial contamination in raw NGS reads.

### CODE REVIEW

**Comment by lowandrew** [comment]
@bioconda/core 

This is both a new package and my first recipe, please review. Thanks!

File: recipes/confindr/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install olctools && $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (rvalieris) [inline recipes/confindr/build.sh]: what is olctools ? if its bioinformatics related you can add it to bioconda before this recipe.
pip shouldn't be used on conda recipes.

Comment (lowandrew) [reply recipes/confindr/build.sh]: My apologies, I didn't realize pip shouldn't be used in conda recipes.

olctools is a package that mostly is completely irrelevant - I've integrated the parts of it that were actually necessary into the ConFindr, so I believe it should now be good to go. Let me know if any other changes need to be made.

Thanks!

File: recipes/confindr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - setuptools
+    - biopython
+    - pysam
+    - pytest
+    - bbmap
+    - jellyfish
+    - blast
+    - mash >=2
+    - samtools
+    - pip
```
Comment (rvalieris) [inline recipes/confindr/meta.yaml]: can you remove this line ?

Comment (lowandrew) [reply recipes/confindr/meta.yaml]: Line has been removed in most recent commit

Comment (rvalieris) [reply recipes/confindr/meta.yaml]: thanks, looks good now, I will merge when the tests finish.

Comment (lowandrew) [reply recipes/confindr/meta.yaml]: Perfect, thanks!

**Pull request merged**